src/Extension.hx

```diff
@@ -16,8 +16,8 @@ class Extension {
 
 			var hlVersion:String = js.node.ChildProcess.execSync('$hl --version');
 			if(hlVersion <= "1.11.0") {
-				final visitButton = "Get from GitHub";
-				Vscode.window.showErrorMessage('Your version of Hashlink (${hlVersion}) does not support debugging on Mac. Install a newer version from here:', visitButton).then(function(choice) {
+				final visitButton:String = "Get from GitHub";
```

Comment:
Huh, what compilation issue does this fix?

Reply:
```
haxe build.hxml
src/Extension.hx:20: characters 5-163 Compiler failure: Unexpected unification error
src/Extension.hx:20: characters 5-163 Please submit an issue at https://github.com/HaxeFoundation/haxe/issues/new
src/Extension.hx:20: characters 5-163 Attach the expression example and the following information:
Haxe: 4.2.1+bf9ff69; OS type: unix;
File "src/typing/callUnification.ml", line 127, characters 53-60
Called from file "src/typing/callUnification.ml", line 127, characters 9-60
Called from file "src/typing/callUnification.ml", line 194, characters 9-21
Called from file "src/typing/callUnification.ml", line 216, characters 14-26
Called from file "src/typing/callUnification.ml", line 321, characters 5-74
Called from file "src/typing/callUnification.ml", line 354, characters 21-41
Called from file "src/typing/callUnification.ml", line 368, characters 31-46
Called from file "src/typing/callUnification.ml", line 400, characters 28-52
Called from file "src/typing/callUnification.ml", line 451, characters 12-62
Called from file "src/typing/typer.ml", line 596, characters 9-51
Called from file "src/typing/typer.ml", line 543, characters 11-50
Called from file "src/typing/typer.ml", line 546, characters 1-30
Called from file "src/typing/typer.ml", line 1543, characters 9-118
Called from file "src/typing/typer.ml", line 1564, characters 10-54
Called from file "src/typing/typer.ml", line 753, characters 27-94
Called from file "src/typing/typer.ml", line 756, characters 9-19
Called from file "src/typing/typer.ml", line 1685, characters 10-38
Called from file "src/typing/typer.ml", line 1453, characters 10-56
Called from file "src/typing/typer.ml", line 753, characters 27-94
Called from file "src/typing/typer.ml", line 756, characters 9-19
Called from file "src/typing/typer.ml", line 1685, characters 10-38
File "src/core/globals.ml", line 117, characters 1-7: Assertion failed
```

Reply:
The API looks like this: https://code.visualstudio.com/api/references/vscode-api#window.showErrorMessage

Reply:
Oh you mean the `:String`? Probably nothing, just a left-over from trying to understand the issue

Reply:
Wow, an assertion failure? Is there a Haxe issue for this already?

Reply:
I didn't create one, feel free to do so. I just assumed that the API call was wrong, since it didn't expect a String at that place

Reply:
I see there are two options, in VSCode's API. How would Haxe handle various overloaded functions with rest parameters?

Reply:
>I didn't create one, feel free to do so. I just assumed that the API call was wrong, since it didn't expect a String at that place

An assertion failure is pretty much always a compiler bug. It even tells you to "please submit an issue". ;)

>I see there are two options, in VSCode's API. How would Haxe handle various overloaded functions with rest parameters?

I haven't looked into the actual issue, but the fix could even just be for Haxe to report a proper error message. An assertion failure simply shouldn't happen.